### PR TITLE
Introduce easyinstall options for installing stand-alone rascsi

### PR DESCRIPTION
**Pull request by rdmark**
- Introduce options for doing everything except installing a systemd service for rascsi/rascsi-web
- Refactored the script to make it more modular
- Added code comments describing each function

### CODE REVIEW

**Review by uweseimet — COMMENTED** [review]
I'm sorry in case I am too persistent, but it would be good if the script outlined which part of the system configuration (e.g. systemd configuration, webserver configuration) it is going to change. Just saying it will do something with the system configuration is more or less implicit when root permissions are required.
The information displayed should enable the user to decide whether he wants these parts of the configuration to be changed or not.

**Comment by uweseimet** [comment]
@rdmark Your latest changes make the installation changes much more transparent, thank you.

I noticed that the manpages are currently install in /usr/share/man. I would have expected /usr/local/man instead, which would be consistent with the installation path for the binaries. (The manpage installation change is currently not yet mentioned, by the way.)

**Comment by rdmark** [comment]
@uweseimet I agree it's good manners by a script to be up-front about potentially destructive operations that it will do, rather than just say "trust us" ;)

**Comment by rdmark** [comment]
@uweseimet Added manpage notices. The manpage installation script is in the Makefile I believe.

**Comment by uweseimet** [comment]
@rdmark Yes, the manpage path is hardcoded in the Makefile. @akuker Any idea why the manpages are installed in /usr/share/man instead of /usr/local/man?

**Comment by uweseimet** [comment]
Just noticed that /usr/local/man is a link to /usr/share/man, i.e. we should update the path in the Makefile to /usr/local/man for consistency. In practice the final manpage location will stay the same.

**Comment by uweseimet** [comment]
@rdmark In case https://github.com/akuker/RASCSI/pull/474 is approved, the easyinstall message about where the manpages are installed should be updated to /usr/local/man. This might be a link, or maybe not, or may even point to different locations, depending on the OS. Looking at /usr/local/man on the respective platform will show.

**Comment by akuker** [comment]
> @rdmark Yes, the manpage path is hardcoded in the Makefile. @akuker Any idea why the manpages are installed in /usr/share/man instead of /usr/local/man?

There is a very logical reason...... because I'm a n00b. I probably just grabbed the first result from google of "where do man files go" and didn't think about what I was doing. 

It really does belong in /usr/local/man

File: easyinstall.sh [hunk 1]
```diff
@@ -94,36 +90,19 @@ function installRaScsi() {
...
-" >> /etc/sudoers'
-    else
-        echo "The sudoers file is already modified for rascsi-web."
-    fi
-
-    sudo systemctl daemon-reload
-    sudo systemctl restart rsyslog
-    sudo systemctl enable rascsi # optional - start rascsi at boot
-    sudo systemctl start rascsi
+    ( make clean && EXTRA_FLAGS="$COMPILER_FLAGS" make -j "${CORES-1}" all CONNECT_TYPE="${CONNECT_TYPE-FULLSPEC}" ) </dev/null
```
Comment (akuker) [inline easyinstall.sh:93]: I think `${CORES-1}` should be `${CORES:-1}`. Please update that throughout.

Comment (rdmark) [reply easyinstall.sh]: Fixed & pushed

**Comment by rdmark** [comment]
> 
> 
> @rdmark In case #474 is approved, the easyinstall message about where the manpages are installed should be updated to /usr/local/man. This might be a link, or maybe not, or may even point to different locations, depending on the OS. Looking at /usr/local/man on the respective platform will show.

Fixed & pushed

**Review by akuker — APPROVED** [review]

**Pull request merged**